local/include/core/proxy_protocol_util.h

```diff
@@ -0,0 +1,118 @@
+/** @file
+ * Common data structures and definitions for Proxy Verifier tools.
+ *
+ * Copyright 2023, Verizon Media
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "swoc/Errata.h"
+#include "swoc/BufferWriter.h"
+#include "swoc/TextView.h"
+#include "swoc/swoc_ip.h"
+
+// PROXY header version
```

Comment:
In this file, these comments can be converted to doxygen comments by using `///` rather than `//`.